xformers/__init__.py

```diff
@@ -8,7 +8,7 @@
 import torch
 
 # Please update the doc version in docs/source/conf.py as well.
-__version__ = "0.0.10"
+__version__ = "0.0.11.dev"
```

Comment:
nit, but I think that it's nice if our current main is not numbered like the pip package, so that we can dissociate in between the two